src/frontend/components/Chat/StudentChat/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { Box } from 'grommet';
+import React from 'react';
+import { ChatConversationDisplayer } from '../SharedChatComponents/ChatConversationDisplayer';
+import { ChatInputBar } from '../SharedChatComponents/ChatInputBar';
```

Comment:
ditto path
```suggestion
import { ChatConversationDisplayer } from '../SharedChatComponents/ChatConversationDisplayer';
import { ChatInputBar } from '../SharedChatComponents/ChatInputBar';
```

Reply:
fix in [30417701d10c51da927b5c824ee5a16a5259c4a5](https://github.com/openfun/marsha/pull/1247/commits/30417701d10c51da927b5c824ee5a16a5259c4a5)